daffodil-core/src/main/scala/org/apache/daffodil/dsom/Term.scala

```diff
@@ -18,16 +18,19 @@
 package org.apache.daffodil.dsom
 
 import java.util.UUID
+
 import org.apache.daffodil.exceptions.Assert
 import org.apache.daffodil.grammar.TermGrammarMixin
 import org.apache.daffodil.schema.annotation.props.gen.YesNo
-import java.lang.{ Integer => JInt }
+import java.lang.{Integer => JInt}
```

Comment:
The style we standardized on is to use spaces, though I think that's not the default in Eclipse. There should be a setting to change this though.

Reply:
Sorry, I'm not sure why IntelliJ changed this!  I'll change it back in the next commit - do you all have a styling XML file that I can use for ScalaStyle or a similar tool?

Reply:
I don't use IntelliJ, but here's our page on IntelliJ setup, Mike added some comments about some style changes:

https://cwiki.apache.org/confluence/display/DAFFODIL/IntelliJ+IDEA+Setup